app/models/barbeque/job_queue.rb

```diff
@@ -2,6 +2,8 @@ class Barbeque::JobQueue < Barbeque::ApplicationRecord
   SQS_NAME_PREFIX = ENV['BARBEQUE_SQS_NAME_PREFIX'] || 'Barbeque-'
   SQS_NAME_MAX_LENGTH = 80
 
+  has_many :sns_subscriptions, class_name: :SNSSubscription, dependent: :destroy
```

Comment:
Umm, I've never seen specifying class_name as Symbol. It's normally String [like this](https://github.com/rails/rails/blob/93982de132f5e3fdcd7794d697a6566bd98da393/activerecord/lib/active_record/associations.rb#L1826-L1827).

It's constantized thing (in somewhere in Rails, not sure) and constantize is defined to String. Thus using Symbol for class_name is just additional cost.